docs/index.md

```diff
@@ -3,17 +3,17 @@
   title: ""
 ---
 
-# AddressBook Level-3
+# ContactCS
 
 [![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/se-edu/addressbook-level3/actions)
 [![codecov](https://codecov.io/gh/se-edu/addressbook-level3/branch/master/graph/badge.svg)](https://codecov.io/gh/se-edu/addressbook-level3)
 
 ![Ui](images/Ui.png)
 
-**AddressBook is a desktop application for managing your contact details.** While it has a GUI, most of the user interactions happen using a CLI (Command Line Interface).
+**ContactCS is a desktop application for managing your contact details.** While it has a GUI, most of the user interactions happen using a CLI (Command Line Interface).
```

Comment:
Same suggestion here. Perhaps it would be better if we include our specific target user?

Reply:
Sure, will do 👍

Reply:
Actually I realized that this part is handled better in Wei Seng's PR #43 (see my comments), hence I think there's no need to change it here and we can just wrap things up later when resolving merge conflicts